File: src/main/java/br/com/test/model/classes/testeTudo.java
package br.com.test.model.classes;

import br.com.model.entities.classes.ItemPedido;

public class TesteTudo {

    public static void main(String[] args) {

        var p = TestePedido.novoPedido();

        for (ItemPedido itemPedido : p.getItensPedido()) {
            itemPedido.setPedido(p);
            itemPedido.getProduto().setListaProdutoFornecedores(TesteProdutoFornecedor.novaListaProdutosFornecedor(TesteFornecedor.novoFornecedor()));
            itemPedido.setAvaliacao(TesteAvaliacao.novaAvaliacao());
        }

        System.out.println(p.toString());
    }
}

File: src/test/java/br/com/model/entities/classes/TesteTudo.kt
package br.com.model.entities.classes

import br.com.newObjects.model.entities.classes.NovoPedido
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class TesteTudo {
    @Test
    fun testeTudo() {
        println(NovoPedido.novoPedido().toString())
    }
}
